cmd/kubermatic-api/swagger.json

```diff
@@ -11290,6 +11290,13 @@
         ],
         "operationId": "listAWSSizesNoCredentialsV2",
         "parameters": [
+          {
+            "type": "string",
+            "x-go-name": "Architecture",
+            "description": "architecture query parameter. Supports: arm64 and x64 types.",
```

Comment:
NIT: try running swagger generation again, sometimes this gets messed up for some reason :(. But 90% of times it resolves as was before.